### PR TITLE
feat (#19): Get current user session

### LINKED ISSUE #19: [Timers API] Get current user session
Take a look at [TimersSessionApi](https://github.com/timemates/sdk/blob/master/sdk/src/commonMain/kotlin/io/timemates/sdk/timers/sessions/TimersSessionsApi.kt) (you should implement this request there).

Look for examples, also, don't forget to implement request [here](https://github.com/timemates/sdk/blob/master/grpc-engine/src/main/kotlin/io/timemates/api/grpc/GrpcTimeMatesRequestsEngine.kt#L115) (add your request and mapping here).

### PR DESCRIPTION
**Pull request by y9slavabogu**
close #19

### CODE REVIEW

**Comment by y9slavabogu** [comment]
open#19

**Review by y9vad9 — CHANGES_REQUESTED** [review]
you miss protofiles

File: grpc-engine/src/main/proto/io/timemates/api/timers/sessions/requests/GetUserSessionsRequest.proto [hunk 1]
```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+import "io/timemates/api/timers/types/Timer.proto";
+
+option java_package = "io.timemates.api.timers.requests";
+
+message GetCurrentTimerSessionRequest {
+    message Response {
+        Timer timer = 1;
```
Comment (y9vad9) [inline grpc-engine/src/main/proto/io/timemates/api/timers/sessions/requests/GetUserSessionsRequest.proto:9]: add optional here:
`optional Timer timer = 1;`

**Review by y9vad9 — CHANGES_REQUESTED** [review]

**Review by y9vad9 — APPROVED** [review]

File: grpc-engine/src/main/proto/io/timemates/api/timers/sessions/TimerSessionsService.proto [hunk 2]
```diff
@@ -67,5 +67,5 @@ service TimerSessionsService {
   /**
   * Gets current timer session
   */
-  rpc getCurrentTimerSession(google.protobuf.Empty) returns (Timer);
+  rpc getCurrentTimerSession(GetCurrentTimerSessionRequest) returns (Timer);
```
Comment (y9vad9) [inline grpc-engine/src/main/proto/io/timemates/api/timers/sessions/TimerSessionsService.proto:70]: there should be google.protobuf.Empty

**Review by y9vad9 — CHANGES_REQUESTED** [review]

**Review by sliderzxc — COMMENTED** [review]
good job

**Review by sliderzxc — APPROVED** [review]

**Pull request merged**